HLTrigger/Configuration/python/customizeHLTforAlpaka.py

```diff
@@ -1,7 +1,554 @@
+        clusterThreshold_layer1 = cms.int32(4000),
+        clusterThreshold_otherLayers = cms.int32(4000),
+        VCaltoElectronGain      = cms.double(1),  # all gains=1, pedestals=0
+        VCaltoElectronGain_L1   = cms.double(1),
+        VCaltoElectronOffset    = cms.double(0),
+        VCaltoElectronOffset_L1 = cms.double(0),
+        InputLabel = cms.InputTag('rawDataCollector'),
+        Regions = cms.PSet(
+            inputs = cms.optional.VInputTag,
+            deltaPhi = cms.optional.vdouble,
+            maxZ = cms.optional.vdouble,
+            beamSpot = cms.optional.InputTag
+        ),
+        CablingMapLabel = cms.string(''),
+        # autoselect the alpaka backend
+        alpaka = cms.untracked.PSet(
+            backend = cms.untracked.string('')
+        )
+    )
+
+    process.hltSiPixelClusters = cms.EDProducer('SiPixelDigisClustersFromSoAAlpakaPhase1',
+        src = cms.InputTag('hltSiPixelClustersSoA'),
+        clusterThreshold_layer1 = cms.int32(4000),
+        clusterThreshold_otherLayers = cms.int32(4000),
+        produceDigis = cms.bool(False),
+        storeDigis = cms.bool(False)
+    )
+
+    process.hltSiPixelClustersCache = cms.EDProducer('SiPixelClusterShapeCacheProducer',
+        src = cms.InputTag( 'hltSiPixelClusters' ),
+        onDemand = cms.bool( False )
+    )
+
+    # legacy EDProducer
+    # consumes
+    #  - SiPixelDigiErrorsHost
+    #  - SiPixelFormatterErrors
+    # produces
+    #  - edm::DetSetVector<SiPixelRawDataError>
+    #  - DetIdCollection
+    #  - DetIdCollection, 'UserErrorModules'
+    #  - edmNew::DetSetVector<PixelFEDChannel>
+    process.hltSiPixelDigis = cms.EDProducer('SiPixelDigiErrorsFromSoAAlpaka',
+        digiErrorSoASrc = cms.InputTag('hltSiPixelClustersSoA'),
+        fmtErrorsSoASrc = cms.InputTag('hltSiPixelClustersSoA'),
+        CablingMapLabel = cms.string(''),
+        UsePhase1 = cms.bool(True),
+        ErrorList = cms.vint32(29),
+        UserErrorList = cms.vint32(40)
+    )
+
+    # alpaka EDProducer
+    # consumes
+    #  - BeamSpotDevice
+    #  - SiPixelClustersSoA
+    #  - SiPixelDigisSoACollection
+    # produces
+    #  - TrackingRecHitsSoACollection<TrackerTraits>
+    process.hltSiPixelRecHitsSoA = cms.EDProducer('SiPixelRecHitAlpakaPhase1@alpaka',
+        beamSpot = cms.InputTag('hltOnlineBeamSpotDevice'),
+        src = cms.InputTag('hltSiPixelClustersSoA'),
+        CPE = cms.string('PixelCPEFastParams'),
+        mightGet = cms.optional.untracked.vstring,
+        # autoselect the alpaka backend
+        alpaka = cms.untracked.PSet(
+            backend = cms.untracked.string('')
+        )
+    )
+
+    process.hltSiPixelRecHits = cms.EDProducer('SiPixelRecHitFromSoAAlpakaPhase1',
+        pixelRecHitSrc = cms.InputTag('hltSiPixelRecHitsSoA'),
+        src = cms.InputTag('hltSiPixelClusters'),
+    )
+
+    ###
+    ### Task: Pixel Local Reconstruction
+    ###
+    process.HLTDoLocalPixelTask = cms.ConditionalTask(
```

Comment:
I thought we were to get rid of ConditionalTasks - see 'minutes' posted here: https://indico.cern.ch/event/1345902/

Reply:
Well yes, but the original ones are still `ConditionalTask`(s) in any HLT menu included in `cms-sw` (see [here](https://cmssdt.cern.ch/dxr/CMSSW/search?q=%22HLTDoLocalPixelTask+%3D%22&case=true&redirect=true)). So I don't know if we should change them here.

Reply:
FWIW, I agree we should take care of these centrally from the TSG side.

Reply:
ok